sdk/rust/oak_app_build/src/main.rs

```diff
@@ -233,6 +263,28 @@ async fn main() -> anyhow::Result<()> {
         modules.insert(name.to_string(), loaded_module);
     }
 
+    // Load signatures.
+    let mut signatures = Vec::new();
+    for location in manifest.signature_manifests.iter() {
```

Comment:
Should the variable be called `manifest_file` instead? Since it may be confusing with `signature_location`

Reply:
Renamed the variable to `signature_manifest_location` to make the intention clearer.